### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by ItsNyoty** [community/events#32] [Community]
Will people who followed the event online get a special badge for their account?

**Comment by queenofcorgis** [comment]
Thanks for asking and thanks for joining us at Universe! As far as I'm aware, there isn't an achievement or badge associated with Universe 2022.

**Reply by ItsNyoty** [reply]
Alright, thanks for answering.

**Reply by mrdiamonddirt** [reply]
ahh thats a shame would of loved a neon octocat on my profile😊
